src/Compilers/CSharp/Portable/Syntax/LookupPosition.cs

```diff
@@ -289,7 +290,8 @@ private static SyntaxToken GetFirstIncludedToken(StatementSyntax statement)
                 case SyntaxKind.FixedStatement:
                     return ((FixedStatementSyntax)statement).FixedKeyword;
                 case SyntaxKind.ForEachStatement:
-                    return ((ForEachStatementSyntax)statement).OpenParenToken.GetNextToken();
+                case SyntaxKind.ForEachComponentStatement:
```

Comment:
I will also look for uses of ForEachStatementSyntax to ensure we handle ForEachComponentSyntax (or the common base).

---
In reply to: [73618167](https://github.com/dotnet/roslyn/pull/12917#discussion_r73618167) [](ancestors = 73618167,73614661)